applications/nrf5340_audio/src/audio/sw_codec_select.c

```diff
@@ -198,6 +218,68 @@ int sw_codec_decode(uint8_t const *const encoded_data, size_t encoded_size, bool
 	return 0;
 }
 
+#ifdef TEST_LC3_DECODER_MODULE
+/**
+ * @brief Dynamic allocation of the memory for the LC3 decoder module.
+ *
+ */
+static int dynamic_memory_allocate(struct amod_parameters *parameters,
```

Comment:
Could this be a helper function in a more general location?

Reply:
Sorry that was me thinking how I would use this in the actual code, it shouldn't be in the PR.